src/tapis-ui/_common/Collapse/Collapse.tsx

```diff
@@ -17,29 +18,41 @@ const Collapse: React.FC<CollapseProperties> = ({
   open,
   className,
   children,
+  isCollapsable,
 }) => {
   const [isOpen, setIsOpen] = useState<boolean>(open ?? false);
   const toggle = useCallback(() => {
     setIsOpen(!isOpen);
   }, [isOpen, setIsOpen]);
 
+  isCollapsable = isCollapsable ? isCollapsable !== undefined : false;
+
+  // Only render collapsable UI if isCollapsable defined and true
   return (
     <div className={className}>
       <div className={styles.header}>
         <div className={styles.title}>{title}</div>
         <div className={styles.controls}>
           <div>{note ?? ''}</div>
-          <Button
-            color="link"
-            className={styles.expand}
-            size="sm"
-            onClick={toggle}
-          >
-            <Icon name={isOpen ? 'collapse' : 'expand'} />
-          </Button>
+          {!isCollapsable ? (
```

Comment:
Instead of conditionally rendering the button, maybe just set it to be disabled if it's not collapsable

Reply:
Or if you don't want to render the button, rather than use ternary:

```
{ isCollapsable && <Button /> }
```

Reply:
I planned on conditionally rendering the button if isCollapsable. It makes more sense the user if there is UI for collapsing when collapsing functionality is unavailable.